src/test/model_service_test.cpp

```diff
@@ -33,46 +33,88 @@
 
 using namespace ovms;
 
-TEST(ModelService, config_reload) {
-    ModelServiceImpl s;
+using TFSGetModelStatusRequest = tensorflow::serving::GetModelStatusRequest;
```

Comment:
We might want to add it into test_utils.hpp, the same as we did with Infer types so it can be used in multiple test files

Reply:
Or we can leave it here if it will never be used outside this file

Reply:
I doubt it will be used outside